packages/iTwinUI-react/src/core/RadioTiles/RadioTile.test.tsx

```diff
@@ -10,40 +10,39 @@ import SvgPlaceholder from '@itwin/itwinui-icons-react/cjs/icons/Placeholder';
 
 it('should render empty radio tile', () => {
   const { container } = render(<RadioTile />);
-  expect(container.querySelector('label > .iui-radio-tile')).toBeTruthy();
-  expect(container.querySelector('.iui-label')).toBeNull();
-  expect(container.querySelector('.iui-description')).toBeNull();
-  expect(container.querySelector('.iui-icon')).toBeNull();
-  expect(container.querySelector('.iui-checkmark')).toBeTruthy();
+  expect(container.querySelector('.iui-radio-tile')).toBeTruthy();
+  expect(container.querySelector('.iui-radio-tile-label')).toBeNull();
+  expect(container.querySelector('.iui-radio-tile-sublabel')).toBeNull();
+  expect(container.querySelector('.iui-radio-tile-icon')).toBeNull();
 });
 
 it('should render radio tile with label', () => {
   const { container } = render(<RadioTile label='My label' />);
-  expect(container.querySelector('label > .iui-radio-tile')).toBeTruthy();
-  const label = container.querySelector('.iui-label') as HTMLElement;
+  expect(container.querySelector('.iui-radio-tile')).toBeTruthy();
+  const label = container.querySelector('.iui-radio-tile-label') as HTMLElement;
   expect(label).toBeTruthy();
   expect(label.textContent).toBe('My label');
 });
 
-it('should render radio tile with label and description', () => {
+it('should render radio tile with label and sublabel', () => {
```

Comment:
Prop is `description` so could just left "description" everywhere

Reply:
Changed to description

Reply:
I wanted that you revert all your renames, just leave "description" as it was before.